src/analysis/buildTypeWeights.ts

```diff
@@ -191,6 +224,8 @@ function parseTypes(
                     fields: {},
                     weight: typeWeights.object || DEFAULT_OBJECT_WEIGHT,
                 };
+            } else {
+                // ? what else can get through here
```

Comment:
Possible input types once we handle mutations. Not sure if/how we would need to handle those

Reply:
I've added note for this but won't address now. My experience is that `inputTypes` are like arguments for the field and will be need to be handled with lists.

Reply:
Agreed, that's my understanding as well. Come back to this in #16